src/main/java/com/educare/unitylend/service/BorrowRequestService.java

```diff
@@ -10,11 +10,14 @@
 public interface BorrowRequestService {
     Boolean createBorrowRequest(BorrowRequest borrowRequest) throws ServiceException;
     List<BorrowRequest> getBorrowRequestForUserId(String userId) throws ServiceException;
-    List<BorrowRequest> getBorrowRequestForCommunity(String communityId) throws ServiceException;
-    Boolean updateEMIDefaults() throws ServiceException;
-    Boolean updateBorrowRequestStatus(Status status) throws ServiceException;
-    List<BorrowRequest> getBorrowRequestsInCommunityLessThanAmount(BigDecimal maxAmount) throws ServiceException;
-    List<BorrowRequest> getBorrowRequestsInCommunityGreaterThanAmount(BigDecimal minAmount) throws ServiceException;
-    List<BorrowRequest> getBorrowRequestsInCommunityInRange(BigDecimal minAmount, BigDecimal maxAmount) throws ServiceException;
+    Boolean updateBorrowRequestStatus(BorrowRequest borrowRequest, Status status) throws ServiceException;
+    List<BorrowRequest> getBorrowRequestsInCommunityLessThanAmount(BigDecimal maxAmount, String communityId) throws ServiceException;
+    List<BorrowRequest> getBorrowRequestsInCommunityGreaterThanAmount(BigDecimal minAmount, String communityId) throws ServiceException;
+    List<BorrowRequest> getBorrowRequestsInCommunityInRange(BigDecimal minAmount, BigDecimal maxAmount, String communityId) throws ServiceException;
     List<BorrowRequest> getAllBorrowRequests() throws ServiceException;
+    boolean validateBorrowRequest(BorrowRequest borrowRequest) throws ServiceException;
+    boolean isUserPartOfCommunity(BorrowRequest borrowRequest) throws ServiceException;
+    boolean isBorrowRequestPending(BorrowRequest borrowRequest) throws ServiceException;
+    boolean isAnythingNull(BorrowRequest borrowRequest) throws ServiceException;
```

Comment:
This method should be part of validateBorrowRequest

Reply:
Due to the reason provided in the previous response in the controller layer, I do not find it appropriate to combine them into a single one.